charts/operator-wandb/charts/weave/templates/deployment.yaml

```diff
@@ -53,16 +53,14 @@ spec:
           env:
             - name: ONLY_SERVICE
               value: weave
-            - name: WANDB_BASE_URL
-              value: http://{{ include "weave.appFullname" . }}:8080
             - name: WANDB_PUBLIC_BASE_URL
               value: {{ .Values.global.host }}
             - name: WEAVE_LOG_FORMAT
               value: json
             - name: WEAVE_LOCAL_ARTIFACT_DIR
               value: /vol/weave/cache
-            - name: WEAVE_AUTH_GRAPHQL_URL
```

Comment:
Fixing this while we're here.  This env var was deprecated a long time ago and this was not really doing anything.  We get the desired effect by setting `WANDB_BASE_URL`.  This may not be correct for all k8s providers